openslides/motions/serializers.py

```diff
@@ -58,19 +57,20 @@ class Meta:
             'versioning',
             'leave_old_version_active',
             'dont_set_identifier',
-            'next_states',)
+            'next_states',
+            'workflow')
```

Comment:
Yes it is redundant. I did not find another way in JSData without this redundant field.